app/controllers/reviews_controller.rb

```diff
@@ -1,20 +1,53 @@
 class ReviewsController < ApplicationController
+  before_action :authenticate_user!, except: [:index, :show]
+
+  def index
+    @reviews = Review.all.limit(10)
+  end
+
   def new
+    @sitcom = Sitcom.find(params[:sitcom_id])
     @review = Review.new
   end
 
   def create
+    @sitcom = Sitcom.find(params[:sitcom_id])
     @review = current_user.reviews.new(review_params)
+    @review.sitcom_id = @sitcom.id
```

Comment:
So you don't want to associate by the id, you want to associate the actual objects:
```ruby
@review.sitcom = @sitcom
```